test/intest1.F90

```diff
@@ -8,6 +8,7 @@ program intest1
   implicit none
   integer mxbfd4, mxds3, nds3, ierme, imgdt
   integer ierndv, iernds, mxr8pm, mxr8lv, iertgp, nr8lv
+  integer l0, l1, l2, l3, l4, l5
   integer*4 mxbf, nbyt, ierr
   integer*4 iupbs01, iupbs3, ireadsb, ibfms
```

Comment:
Thanks Alex, and just one last request - can we change the variable names to "len0", "len1",...,"len5", rather than "l0, l1,...,l5"?  Sorry, but every time I look at this I'm reading integers 10, 11,...,15, b/c the 1 looks too much like the lowercase l.

Otherwise, looks good - and thanks!